config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json

```diff
@@ -56,25 +71,11 @@
                 "delegate": {
                   "comment":"resolve access tokens and retrieve metadata about the token. The endpoint typically returns the time until the token expires, the OAuth 2.0 scopes associated with the token, and potentially other information",
```

Comment:
can you update (or remove) the comment please, as it relates to TokenIntrospectionAccessTokenResolver

Reply:
Removed the comments since the resolver is straight forward and comment was outdated.